mno_analysis_tools/docker-run-compute-messages-per-period.sh

```diff
@@ -41,7 +41,7 @@ docker build --build-arg INSTALL_MEMORY_PROFILER="$PROFILE_MEMORY" -t "$IMAGE_NA
 
 # Create a container from the image that was just built.
 if [[ "$PROFILE_MEMORY" = true ]]; then
-    PROFILE_MEMORY_CMD="mprof run -o /data/memory.prof"
+    PROFILE_MEMORY_CMD="mprof run -o /data/memory_3.prof"
```

Comment:
please rename the in respect to the script names

Reply:
Updated thanks